web/thesauruses/java/15/operators.json

```diff
@@ -106,6 +106,24 @@
       "not-implemented": "true",
       "name": "Is not operator"
     },
+    "logical_and": {
```

Comment:
You added operand placement in Java 11's file but not 15 and 17. Could you do the same for those as well?